platform/ios/resources/sv.lproj/Localizable.strings

```diff
@@ -31,18 +34,33 @@
 /* Accessibility label */
 "INFO_A11Y_LABEL" = "Om den här kartan";
 
+/* User-friendly error description */
+"LOAD_MAP_FAILED_DESC" = "Kartan kunde inte laddas på grund av att ett okänt fel inträffade.";
+
+/* User-friendly error description */
+"LOAD_STYLE_FAILED_DESC" = "The map failed to load because the style can't be loaded.";
+
 /* Accessibility label */
 "LOGO_A11Y_LABEL" = "Mapbox";
 
 /* Accessibility label */
 "MAP_A11Y_LABEL" = "Karta";
 
 /* Map accessibility value */
-"MAP_A11Y_VALUE" = "Zoom %1$dx\n%2$ld annotation(s) visible";
+"MAP_A11Y_VALUE" = "Zoom %1$dx\n%2$ld annotering(ar) synliga";
```

Comment:
@frederoni, would you mind adding a .stringsdict file indicating the different grammatical variations of this string? (If you’re unfamiliar with [the stringsdict format](https://developer.apple.com/library/content/documentation/MacOSX/Conceptual/BPInternational/StringsdictFileFormat/StringsdictFileFormat.html), feel free to provide the necessary information here and I’ll add it in.